src/tests/ftest/pool/evict_test.py

```diff
@@ -146,43 +151,46 @@ def test_evict(self):
         by the evict
         :avocado: tags=all,pr,daily_regression,full_regression
         :avocado: tags=small
-        :avocado: tags=pool,poolevict
+        :avocado: tags=pool,pool_evict
         :avocado: tags=DAOS_5610
         """
-        pool = []
+        self.pool = []
         container = []
         # non_pool_servers = []
+
         # Target list is configured so that the pools are across all servers
         # except the pool under test is created on half of the servers
         pool_tgt = list(range(len(self.hostlist_servers)))
         pool_tgt_ut = list(range(int(len(self.hostlist_servers)/2)))
         tlist = [pool_tgt, pool_tgt, pool_tgt_ut]
         pool_servers = [self.hostlist_servers[:len(tgt)] for tgt in tlist]
         # non_pool_servers = [self.hostlist_servers[len(tgt):] for tgt in tlist]
+
         # Create Connected TestPool
         for count, target_list in enumerate(tlist):
-            pool.append(self.connected_pool(pool_servers[count], target_list))
+            self.connected_pool(pool_servers[count], target_list)
             # Commented out due to DAOS-3836.
             # if len(non_pool_servers[count]) > 0:
             #    self.assertFalse(
             #        pool[count].check_files(non_pool_servers[count]),
             #        "Pool # {} data detected on non pool servers {} ".format(
             #            count+1, non_pool_servers[count]))
 
-            self.log.info("Pool # %s is connected with handle %s",
-                          count+1, pool[count].pool.handle.value)
+            self.log.info(
+                "Pool # %s is connected with handle %s", count + 1,
+                self.pool[-1].pool.handle.value)
 
-            container.append(TestContainer(pool[count]))
+            container.append(TestContainer(self.pool[-1]))
             container[count].get_params(self)
             container[count].create()
             container[count].write_objects(target_list[-1])
 
         try:
             self.log.info(
                 "Attempting to evict clients from pool with UUID: %s",
-                pool[-1].uuid)
+                self.pool[-1].uuid)
             # Evict the last pool in the list
-            pool[-1].dmg.pool_evict(pool=pool[-1].pool.get_uuid_str())
+            self.pool[-1].dmg.pool_evict(pool=self.pool[-1].pool.get_uuid_str())
```

Comment:
```suggestion
            pool[-1].dmg.pool_evict(pool=pool[-1].pool.get_uuid_str())
```
Possibly use `TestPool.evict()` instead here - which will raise an `avocado.core.exceptions.TestFail` exception if it fails.

Reply:
I'll do it in later PR because I want to get this in ASAP and I don't want to increase the chance of making a mistake.